.github/workflows/performance-testing.yml

```
@@ -20,7 +20,7 @@ jobs:
 
       - name: Cache pnpm dependencies
         id: cache-pnpm-dependencies
-        uses: actions/cache@v3.4.0
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)